### PR TITLE
FIX Iterations & Spelling

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -30,7 +30,25 @@ def is_valid_operator_usage(user_input, list_of_operators):
 def validate(user_input):
     user_input = user_input.replace(" ", "")  # to get rid of same spaces
     list_of_operators = ["+", "-", "*", "/"]
-    list_of_numbers = list(str(x) for x in range(0, 10))
+    list_of_numbers = list(str(x) for x in range(0, 9))  # 10 is not a digit, digits are between 0-9
+    """
+    When you iterate a string char by char you will never see digit "10" because 10 is not a digit.
```
Comment (UmutStar) [inline controller.py]: When i write like this """ list_of_numbers=list(str(x) for x in range(0,9))""" list_of_numbers will be [0,1,2,3,4,5,6,7,8]. range(0,10)'s mean 0 to up to 10.

Comment (Orkuncakilkaya) [reply controller.py]: Sorry for my misunderstanding :)

Comment (UmutStar) [reply controller.py]: Np it is nice to see someone wrote something to me.

Comment (Orkuncakilkaya) [reply controller.py]: Anytime, excuse my python :D

**Comment by Fabsolute** [comment]
zaa

**Pull request closed without merging**